javascript/javascript3/week3/class.js

```diff
@@ -0,0 +1,79 @@
+// Shopping cart using Classes
+class Product {
+  constructor(name, price) {
```

Comment:
what currency units is this price specified in? The variable naming should reflect that.